tests/test_launcher.py

```diff
@@ -21,6 +21,7 @@
 import shutil
 import time
 from multiprocessing import Event
+from multiprocessing.managers import DictProxy  # type: ignore # noqa: F401
```

Comment:
Do we still need this? since the pickle related issues has been resolved

Reply:
I think it doesn't hurt